libs/documentation/src/lib/storybook/formly/formly-text-area/formly-text-area-description/formly-text-area-description.component.html

```diff
@@ -0,0 +1,10 @@
+<form [formGroup]="form">
+  <formly-form
+    [model]="model"
+    [fields]="fields"
+    [options]="options"
+    [form]="form"
+    (modelChange)="onModelChange($event)"
```

Comment:
Assuming you create another demo for this output, remove this from the HTML in the non-output demos.